packages/suite/src/actions/suite/storageActions.ts

```diff
@@ -52,6 +55,42 @@ export const removeAccountDraft = async (account: Account) => {
 
 // send form drafts end
 
+export const saveFormDraft = async (key: string, draft: FormDraft) => {
+    if (!(await isDBAccessible())) return;
+    return db.addItem('formDrafts', draft, key, true);
+};
+
+export const removeFormDraft = async (key: string) => {
+    if (!(await isDBAccessible())) return;
+    return db.removeItemByPK('formDrafts', key);
+};
+
+export const saveAccountFormDrafts = (accountKey: string) => async (
+    _: Dispatch,
+    getState: GetState,
+) => {
+    if (!(await isDBAccessible())) return;
+
+    const { formDrafts } = getState().wallet;
+
+    await Promise.all(
+        FormDraftPrefixKeyValues.map(prefix => {
+            const formDraftKey = getFormDraftKey(prefix, accountKey);
+            const formDraft = formDrafts[formDraftKey];
+            return formDraft ? db.addItem('formDrafts', formDraft, formDraftKey, true) : null;
+        }),
+    );
+};
+
+export const removeAccountFormDrafts = async (accountKey: string) => {
+    if (!(await isDBAccessible())) return;
+    await Promise.all(
```

Comment:
this should return only promises, without await (like removeAccountDraft) 
`Promise.all` is performed in line 233

Reply:
An async function can return a single Promise. So I changed the function's contract to return the single promise with iteration over form draft prefixes in `forgetDevice`